tests/cypress/integration/pro/subscriptions_spec.js

```diff
@@ -12,6 +14,8 @@ context("/pro/dashboard", () => {
     ];
 
     cy.login();
+    cy.wait(2000);
```

Comment:
Using `cy.wait()` with arbitrary time values is generally not recommended and should be used as a last resort (not reliable, can lead to flaky tests).

Do you know why we need to wait here, are we waiting for a specific network request to complete or UI element to become visible?

We should be able to assert those instead, e.g. use `cy.intercept()` to wait for a specific network request.

Also, any such check should ideally go to the login command to reduce repetition, which you can safely modify.

Reply:
The reason I added this wait statement was because this was the only test that was failing right after the loging was happening. And I noticed that in other similar tests there is a wait statement after the login. But good point with adding a wait statement to login itself 👍

Reply:
It might be useful to run cypress in the UI mode (`yarn cypress:open`)  where you'll be able to see what's actually happening as the runner goes through each step.
https://docs.cypress.io/guides/core-concepts/cypress-app

Reply:
Some suggestions for alternatives to timeout wait: https://filiphric.com/waiting-in-cypress-and-how-to-avoid-it

Reply:
Let's try and pin this down if we can and use alternatives. Otherwise, let's at least make sure `wait` it's only added in a single place (login util).

Reply:
I removed all the `cy.wait()` statements after logins. Instead slightly increased the timeout in the `login` task.